assets/helpers/utilities.js

```diff
@@ -76,6 +76,19 @@ function deserialiseJsonObject(serialised: string): ?Object {
 
 }
 
+/* eslint-disable no-bitwise, no-mixed-operators */
+// generate a universally unique identifier
+function UUID(mask: string = 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx') {
+  const replacer = (c) => {
+    let d = new Date().getTime();
+    const r = (d + (Math.random() * 16)) % 16 | 0;
+    d = Math.floor(d / 16);
+    return (c === 'x' ? r : (r & 0x3 | 0x8)).toString(16);
+  };
+  return mask.replace(/[xy]/g, replacer);
+}
```

Comment:
@rupertbates added a uuid library in #248, can we make use of that here?

Reply:
Yes!